inst/tests/tests.Rraw

```diff
@@ -18439,3 +18439,43 @@ dt = data.table(a = 1L)
 test(2252.1, dt[, b:=2L], error = "\\[ was called on a data.table.*not data.table-aware.*':='")
 test(2252.2, dt[, let(b=2L)], error = "\\[ was called on a data.table.*not data.table-aware.*'let'")
 rm(.datatable.aware)
+
+# tests for trunc.char handling wide characters # 5096
+accented_a = "\u0061\u0301"
+ja_ichi = "\u4E00"
+ja_ni = "\u4E8C"
+ja_ko = "\u3053"
+ja_n = "\u3093"
+dots = "..."
+clean_regex = "^\\d+:\\s+" # removes row numbering from beginning of output
+# Tests for combining character latin a and acute accent, single row
+DT = data.table(strrep(accented_a, 4L))
+test(2253.01, gsub(clean_regex, "", capture.output(print(DT))[-1L]), strrep(accented_a, 4L), options=list(datatable.prettyprint.char = 4L))
```

Comment:
stylistic request before we merge -- use style `test(n, options=, ...)` to make `options=` clear up-front.

Can you also try going back to the simple `output=` approach of the tests? It will make it a lot cleaner to read. Hopefully the simpler test also passes on windows GHA, if not we can revert back to this functioning version.

Reply:
I'll do this once I get home later!

Reply:
For the first few tests, the simplified `output=` works well:

```R
...
test(2253.04, options=list(datatable.prettyprint.char = 4L), DT, output=strrep(ja_ichi, 4L))
test(2253.05, options=list(datatable.prettyprint.char = 3L), DT, output=paste0(strrep(ja_ichi, 3L), dots))
test(2253.06, options=list(datatable.prettyprint.char = 1L), DT, output=paste0(strrep(ja_ichi, 1L), dots))
...
```

But for the later tests (Once we have more columns/rows), I find that using this approach requires something like:
```R
test(2253.07, options=list(datatable.prettyprint.char = 4L), DT, output="     V1\n1:    á\n2:   áá\n3:  ááá\n4: áááá")
```
I'm not a huge fan of how the expected output looks, but if you think this way is more readable I can refactor the rest of the tests. For now, I'll commit the options syntax style change and using the simple approach with the first few tests.

Reply:
ah, I had something else in mind but it looks like the force push erased the history in the other branch. oh well. I won't belabor it any further :)